homeassistant/components/jewish_calendar/binary_sensor.py

```diff
@@ -0,0 +1,74 @@
+"""Support for Jewish Calendar binary sensors."""
+import logging
+
+from homeassistant.components.binary_sensor import BinarySensorDevice
+import homeassistant.util.dt as dt_util
+
+from . import DOMAIN, SENSOR_TYPES
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+    """Set up the Jewish Calendar binary sensor devices."""
+    if discovery_info is None:
+        return
+
+    for sensor, sensor_info in SENSOR_TYPES["binary"].items():
+        async_add_entities(
```

Comment:
Please collect all entities first and make a single call to `async_add_entities`.